src/components/ToggleElementDropdown/ToggleElementDropdown.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import core from 'core';
+
+import selectors from 'selectors';
+import actions from 'actions';
+
+import { zoomTo } from 'helpers/zoom';
+
+import ToggleElementButton from '../ToggleElementButton';
+
+import './ToggleElementDropdown.scss';
+
+class ToggleElementDropdown extends React.PureComponent {
+
+  constructor(props){
+    super(props);
+    this.state = { zoom: Math.ceil(core.getZoom() * 100), value: 100 };
```

Comment:
I'm not quite sure if we need `zoom` as part of the state. I think we can just call `setState` on `value` to trigger a re-render. What do you think?

Reply:
Sounds good!